src/server/common/model/answer/address.js

```diff
@@ -79,6 +79,7 @@ export class Address extends AnswerModel {
 
   get html() {
     return Object.values(this.value ?? [])
+      .filter((line) => line !== undefined)
```

Comment:
The process of unpacking & repacking an object causes: 

```
{ addressLine1: 'fooBar', addressTown: 'London', addressPostcode: 'N11NB' }
```

into: 

```
{ addressLine1: 'fooBar', addressLine2: undefined, addressTown: 'London', addressCounty: undefined, addressPostcode: 'N11NB' }
```

I don't think this matters for us, other than in this context - but happy to be challenged.

NB - the reason I didn't use lodash's `pick` is because it isn't typesafe.